intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,11 +129,15 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify their **state** and their generic **behavior**.
+
+**State**, **Attribute**, **Behavior**, and **Method** are all new terms we haven't heard before, but don't be intimidated by them! 
+
+**State** is just a term that means "all the **attributes** of an object". **Attribute** is simply a specific type of **variable** defined within a class. 
 
-**State** describes any data or attributes that should be kept over its lifetime—from the moment we create an instance until we let the system reclaim its resources.
+These **attributes** will be kept over the object's lifetime-from the moment we create an instance until the object goes out of scope.
```

Comment:
Lifetime and scope are two different concepts.  Objects don't go out of scope. A variable referencing an object goes out of scope. But even then, the lifetime of the object can be much longer.

I prefer the original phrasing of 'until we let the system reclaim its resources,' since it completely sidesteps this quibble.

Also, the original "hyphen" is actually an em dash.  Hard to notice in monospace font, but I restored it in the suggestion here.

```suggestion
These **attributes** will be kept over the object's lifetime—from the moment we create an instance until we let the system reclaim its resources.
```

Reply:
I still don't fully understand but I'm outnumbered so will assume there's just a gap in my understanding that I hope to learn more about and put it back like you suggest!

Reply:
So...maybe I shouldn't have done this but when I totally rewrote that all just now, I just removed that whole part because it felt so out of place here and in my opinion is not going to really register with students in such a brief mention.
Would you agree this is more appropriate to talk about this in the creating-instances lesson and drop it here anyway?